docs/moduleAna5Utils.rst

```diff
@@ -0,0 +1,99 @@
+direct comparison.
+In addition to the **tt-diagram**, :py:mod:`ana5RangeTime` also offers the possibility to
+produce simulated **range-time diagrams** of the flow parameters with respect to a radar's field
+of view. In this manner, simulation results can be directly compared to radar measurements in terms
+of front position and inferred approach velocity. The colorcoding of the simulated
+**range-time** diagrams refers to average values of the chosen flow parameter
+(e.g. flow depth, flow velocity) at specified range gates. This colorcoding is not directly
+comparable to the intensity given in the range-time diagram from radar measurements.
+
+.. Note::
+  The **tt-diagram** and the **range-time diagram** can be produced *during run time* of
+  :py:mod:`com1DFA`, or as a postprocessing step. However, the second option requires saving all the
+  required time steps of the flow parameter fields, which is much more computationally expensive
+  compared to the first option.
+
+To run
+~~~~~~~
+
+During run-time of :py:mod:`com1DFA`:
+
+* in your local copy of ``com1DFA/com1DFACfg.ini`` in [VISUALISATION] set `createRangeTimeDiagram`
+  to True and chose if you want a *TTdiagram* by setting this flag to True or in the case of a
+  simulated range-time diagram to False
+
+* in your local copy of ``ana5Utils/distanceTimeAnalysisCfg.ini`` you can adjust the default settings
+  for the generation of the diagrams
+
+As a postprocessing step:
+
+* have a look at :py:mod:`runScripts.runThalwegTimeDiagram.py` and :py:mod:`runScripts.runRangeTimeDiagram.py`
+
+* in your local copy of ``ana5Utils/distanceTimeAnalysisCfg.ini`` you can adjust the default settings
+  for the generation of the diagrams
+
+The resulting figures can be found in ``avalancheDirectory/Outputs/ana5Utils``.
+
+
+.. figure:: _static/thalwegTime_FD.png
+    :width: 90%
+
+    Thalweg-time diagram example. Maximum approach velocity is computed as the ratio of the
+    distance travelled by the front and the respective time needed for two increments along the path.
+
+
+.. Note::
+  The **tt-diagram** requires info on an avalanche path (see :ref:`moduleAna3AIMEC:ana3AIMEC: Aimec`).
+  The simulated **range-time diagram** requires info on the x, y coordinate of the radar location, a point
+  in the direction of the field of view, the aperture angle and the width of the range gates.
+
+
+
+Theory
+~~~~~~~~~
+
+Thalweg-time diagram
+~~~~~~~~~~~~~~~~~~~~~~
+
+First, the flow parameter result field is transformed into a path-following coordinate system.
+For this step, functions from :py:mod:`ana3AIMEC` are used.
+The distance to the *start of runout area point* is determined and the mean values of the cross
+profiles along the avalanche path are computed.
+In the **tt-diagram**, the avalanche front position is indicated with black dots and the colored
+field indicates the mean values of the flow parameter at cross profiles along the avalanche path
+for each time step.
```

Comment:
for which rasters / flow parameters can a TT diagram be created? all - whats the difference ? is the front always defined with respect to the analyzed aparameter?

Reply:
whort info on the depicted velocities: in which direction are they measured (approach velocities in projected avalanche path), particularly with respect to the flow velocities (surface parallel?). What does the location of the max front velocity indicate?

Reply:
I updated the paragraph